pkg/applicationmanager/controllers/applicationrestore.go

```diff
@@ -1695,6 +1695,22 @@ func (a *ApplicationRestoreController) cleanupRestore(restore *storkapi.Applicat
 			return fmt.Errorf("cancel restore: %s", err)
 		}
 	}
+	// Directly calling DeleteResourceExport with out checking backuplocation type.
+	// For other backuplocation type, expecting Notfound
+	crName := getResourceExportCRName(utils.PrefixRestore, string(restore.UID), restore.Namespace)
```

Comment:
Is the intention to delete from both AdminNamespace and restore.Namespace?
Currently both the deletes are deleting from restore.Namespace

Reply:
Fixed resource's RE also to use restore CR namespace instead of Adminnamespace.